webapp/src/pages/Recipe/Recipe.tsx

```diff
@@ -1,10 +1,82 @@
 import React from 'react';
-import './Recipe.css';
+import { useParams } from 'react-router-dom';
+import { Panel } from 'primereact/panel';
+import { Splitter, SplitterPanel } from 'primereact/splitter';
+// ? Not sure why this library can't be found even though when I console.log(uuidv4()) it works?
+// import { v4 as uuidv4 } from 'uuid';
+import PlaceHolder from '../../assets/images/samosa.jpg';
+import { useQuery } from 'urql';
+import './Recipe.scss';
+// * homemade uuid ^_^ since the libaray won't work!
+const uuid: number = Math.floor(Math.random() * 10000);
```

Comment:
What error was it giving out of curiosity? If it's not a pure JS library I could see it having issues.

Also a UUID often turns into a string representation (from binary data) and requires a specific spec. I see you're just using it as a random ID here though so no big deal